### DISCUSSION TITLE
Bungee Information

### DISCUSSION THREAD

**Discussion by banit** [Zrips/CMI#2560] [Q&A]
Since there is no Bungeecord FAQ it would be really helpful to know wich features will have a Bungeecord Functionality and wich not.
A Simple list would do.

- Economy
- Chat
- Tablist
and so on..

**Comment by mrfloris** [comment]
What I know about cmi + bungee, I've put in this FAQ.
https://github.com/mrfdev/CMI/blob/master/Resources/FAQ/cmi-bungee.md

Hopefully over time more and more CMI features become 'network friendly' and allow syncing.

The solution so far has been to consider moving to MySQL databases, and using the MySQL player-data bridge plugin on spigot made by someone else. https://www.spigotmc.org/resources/mysql-player-data-bridge.8117/
